MassRegistrationOfSAPSystems/RegisterSIDs.ps1

```diff
@@ -1,22 +1,64 @@
-#Run the below commands before using the script
-#Connect-AzAccount -SubscriptionId 'yyyy-yyyy-yyyy-yyyy'
+# ---------------------------------------------------------------------------------- 
+    # Copyright (c) Microsoft Corporation. All rights reserved.
+    # Licensed under the MIT License. See License in the project root for
+    # license information.
+# ----------------------------------------------------------------------------------
 
-# Replace <filePath> to the input CSV file
-$inputfile = "<filePath>"
+<#
 
-# Replace <filePath> to the output CSV file
-$outputfile = "<filePath>"
+.SYNOPSIS
+    Powershell script that register multiple SAP Systems in Azure Center for SAP Solutions Parallely
 
-# Specify the number of seconds to monitor the job status
-$MonitoringIntervalInSeconds = 30
+.DESCRIPTION
+    This script:
+    - fetches the config information from the input CSV file
+    - Create a job for registring each SID and monitor the job status
+    - Provide the status of the execution of each job on console as well as output CSV file
 
-# Specify the maximum number of parallel jobs to run
-$MaxParallelJobs = 10
+.PARAMETER InputFile
+    This is the full path of the csv file which has the inputdata for the SAP systems that are to be registered
 
+.PARAMETER OutputFile
+    This is the full path of the output csv file which will be generated by the script with the status of the registration process.
+
+.PARAMETER MonitoringIntervalInSeconds
+    This specifies the number of seconds for which script will wait to check the status of the jobs during the execution
+
+.PARAMETER MaxParallelJobs
+    This specifies the limit of parallel Jobs that will run at a time
+
+.EXAMPLE
+    C:\MassRegistrationOfSAPSystems\RegisterSIDs.ps1 -InputFile "C:\input\RegistrationData.csv" -OutputFile "C:\output\RegistrationDataOutput.csv"
+
+    To Register multiple SAP Systems in Azure Center for SAP Solutions Parallely with default values of MonitoringIntervalInSeconds and MaxParallelJobs
+
+.EXAMPLE
+    C:\MassRegistrationOfSAPSystems\RegisterSIDs.ps1 -InputFile "C:\input\RegistrationData.csv" -OutputFile "C:\output\RegistrationDataOutput.csv" -MonitoringIntervalInSeconds 30 -MaxParallelJobs 10
+
+    To Register multiple SAP Systems in Azure Center for SAP Solutions Parallely with custom values of MonitoringIntervalInSeconds and MaxParallelJobs 
```

Comment:
Why this statement here? We have already clarified what the script is doing and what are each of the input parameters.

Reply:
Removed the statement